.github/ISSUE_TEMPLATE/Feature.md

```diff
@@ -7,12 +7,19 @@ about: Suggest a new feature or enhancement for rclone
 
 Welcome :-)
 
-So you've got an idea to improve rclone? We love that! You'll be glad to hear we've incorporated hundreds of ideas from contributors already.
+So you've got an idea to improve rclone? We love that!
+You'll be glad to hear we've incorporated hundreds of ideas from contributors already.
+Probably the latest release has your feature. Try to update your rclone, run:
 
-Here is a checklist of things to do:
+    rclone self-update
 
-  1. Please search the old issues first for your idea and +1 or comment on an existing issue if possible.
-  2. Discuss on the forum first: https://forum.rclone.org/
+If your rclone reports `unknown command "self-update"` then you'll need to update manually
+following the install instructions at https://rclone.org/install
+
+If it still isn't there, here is a checklist of things to do:
+
+  1. Discuss on the forum first: https://forum.rclone.org/
+  2. Search the old issues for your idea and +1 or comment on an existing issue if possible.
```

Comment:
2 & 1 should probably be inter-changed.

Reply:
Don't we give priority to the forum now?

Reply:
Yes, we do, but if there is an issue/FR already present, then creating a duplicate post about it won't help.

Reply:
ok. will swap back